src/sparseml/transformers/text_classification.py

```diff
@@ -464,13 +486,27 @@ def main():
     )
 
     # Preprocessing the datasets
+    if data_args.input_column_names is not None:
+        if "," in data_args.input_column_names:
+            # two input columns
+            columns = data_args.input_column_names
```

Comment:
doesn't this need to split input_column_names on ","?

Reply:
yeah, missed this on commit